src/Google.Bigquery.V2/BigqueryResult.cs

```diff
@@ -153,24 +155,25 @@ internal Row(BigqueryResult parent, TableRow rawRow)
                 {
                     object rawValue = RawRow.F[index].V;
                     var field = _parent.Schema.Fields[index];
-                    switch (field.Type)
+                    var type = EnumMap<BigqueryDbType>.ToValue(field.Type);
```

Comment:
Will add an internal extension method for now. We may want to expose it, I agree... let's see.